requirements-dev.txt

```
@@ -0,0 +1,3 @@
+pytest==7.1.0
+pytest-randomly==3.11.0
+flaky==3.7.0
```

See comment in requirements.txt:
```suggestion
flaky==3.7.0
pytest==7.1.0
pytest-randomly==3.11.0
```